pkg/cluster/manager.go

```diff
@@ -1057,6 +1070,15 @@ func (m *Manager) Deploy(
 		base.GlobalOptions.SSHType = sshType
 	}
 
+	if !opt.NoLabels {
+		// Check if TiKV's label set correctly
+		lbs := topo.(*spec.Specification).LocationLabels()
+		kvs := topo.(*spec.Specification).TiKVServers
+		if err := spec.CheckTiKVLocationLabels(lbs, kvs); err != nil {
+			return perrs.Errorf("check TiKV label failed, please fix that before continue:\n%s", err)
```

Comment:
Prefer to attach the label information to the error.

Reply:
the `err` contains label info
![image](https://user-images.githubusercontent.com/8718109/93880506-2cbb5500-fd10-11ea-8857-16d96e9efcc5.png)